### DISCUSSION TITLE
Generator Could Not Start

### DISCUSSION THREAD

**Discussion by ifeanyidotdev** [RobertCraigie/prisma-client-py#501] [General]
i install the package in my virtual env, 

i  initaized create my models, then when i tried runing prisma db pull , i encourted an error saying the generator in the js module could not start 
<img width="814" alt="Screenshot 2022-09-29 at 10 20 05 PM" src="https://user-images.githubusercontent.com/40169444/193144129-b25c7d0b-3dd2-4581-9826-44062ae1efbe.png">

i would need some help on this

**Comment by RobertCraigie** [comment]
Hey @Mremmalex, unfortunately this is a known issue, see https://github.com/prisma/prisma/issues/10564.

While this is an internal Prisma bug I'm hoping to fix this soon with a big internal refactor: #454

In the meantime you can try installing the Python package from the WIP branch like so:

```
pip install -U git+https://github.com/RobertCraigie/prisma-client-py@refactor/remove-pkg-cli
```
